concordium-sdk/src/main/java/com/concordium/sdk/responses/blocksummary/updates/queues/UpdateQueues.java

```diff
@@ -0,0 +1,149 @@
+package com.concordium.sdk.responses.blocksummary.updates.queues;
+
+import com.concordium.sdk.responses.blocksummary.updates.Fraction;
+import com.concordium.sdk.responses.blocksummary.updates.ProtocolUpdate;
+import com.concordium.sdk.responses.blocksummary.updates.chainparameters.PoolParameters;
+import com.concordium.sdk.responses.blocksummary.updates.chainparameters.rewards.GasRewards;
+import com.concordium.sdk.responses.blocksummary.updates.chainparameters.rewards.MintDistribution;
+import com.concordium.sdk.responses.blocksummary.updates.chainparameters.rewards.TransactionFeeDistribution;
+import com.concordium.sdk.responses.blocksummary.updates.keys.Level1KeysUpdates;
+import com.concordium.sdk.responses.blocksummary.updates.keys.Level2KeysUpdates;
+import com.concordium.sdk.responses.blocksummary.updates.keys.RootKeysUpdates;
+import com.concordium.sdk.transactions.CCDAmount;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.ToString;
+
+/**
+ * UpdateQueues must match
+ * https://github.com/Concordium/concordium-base/blob/23c2f4f2fb524e60548f052d0422b567ec977822/haskell-src/Concordium/Types/UpdateQueues.hs#L119
```

Comment:
This is a note for a developer/maintainer, not for the public documentation of the class.